tests/crypto/Kconfig

```diff
@@ -14,4 +14,25 @@ config CRYPTO_LONG_RUNNING_VECTORS_DISABLE
 	  Do a search of CRYPTO_LONG_RUNNING_VECTORS_DISABLE amongst the
 	  test_vectors_* files to see which vectors are affected.
 
+config CRYPTO_LARGE_VECTORS_DISABLE
```

Comment:
Just wondering.
Why not make this opposite.

State what you want to test ?

```
config CRYPTO_TEST_LONG_RUNNING_VECTORS
```

Reply:
Agreed! Negated the style of these.